js/main.js

```diff
@@ -0,0 +1,30 @@
+const $image = document.querySelector('.url');
+const $imgUrlInput = document.querySelector('#photo-url');
+const $titleInput = document.querySelector('#entry-title');
+const $notesInput = document.querySelector('#notes');
+
+$imgUrlInput.addEventListener('input', handleUrlInput);
+
+function handleUrlInput(e) {
+  $image.setAttribute('src', $imgUrlInput.value);
+}
+
+const $entry = document.querySelector('.journal-entry');
+
+$entry.addEventListener('submit', handleSubmission);
```

Comment:
I don't think this is actually working ? Your `div` seems to have the class of `journal-entry`, but you need to listen for `submit` events on your `form`.